compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -126,9 +126,7 @@ cmdIde =
         <$> telemetryOpt
         <*> debugOpt
         <*> enableScenarioOpt
-        <*> optGhcCustomOptions
-        <*> shakeProfilingOpt
-        <*> optional lfVersionOpt
+        <*> optionsParser numProcessors (EnableScenarioService True) (pure Nothing)
```

Comment:
@cocreature Is this too permissive? What's the right place to add tests for this?

Reply:
Should be fine I think. I don’t see any options that are less useful for `damlc ide` than they are for `damlc build`. Not sure what good tests for this would be. We have tests for the config parser but I think the parser is not the issue here. I guess you could extend the LSP tests to check different flag combinations but I’m fine to not have tests for this now.